### ISSUE TITLE
libreoffice command

### ISSUE THREAD

**Issue by SalahSaid** [w3nl/ppt-png#2]
Hi,

I m using the latest version of the ppt-png npm (0.3.0) and the latest of Libreoffice too (6.2.4.2).
The command libreoffice used by ppt-png (file convert.js line 210) does not exist in Libreoffice 6.2.4.2. so I have a  '/bin/sh: libreoffice: command not found ' error. When I replace the libreoffice command by soffice command that works fine.

Is there any any ppt-png modification for this or should I use an old version of LibreOffice.

Thanx for your reply.

Sincerely
Said

**Comment by PieterWigboldus** [comment]
I have released version 0.4.0.
You can now set the convert command in the options.

```
new Converter({
    files: ...,
    ...
    documentConvert: 'libreoffice --headless --convert-to pdf --outdir',
    ...
).run();
```

**Comment by SalahSaid** [comment]
That's greeeeat ! it works fine ! thank you so match

Just a point : I think a space should be added at the end of the documentConvert argument
otherwise the final command will be like 'soffice --headless --convert-to pdf --**outdir/Users/**...' and will fail.

Thanx againt !

**Comment by PieterWigboldus** [comment]
Thanks!
I have fixed it in version 0.4.2

**Comment by SalahSaid** [comment]
That's works fine.
Thanks !
